Elasticsearch Transcript Store component

pratikpanda opened the pull request:
Elasticsearch based transcript store component for bots created using Microsoft Bot Framework.

pratikpanda commented:
@garypretty can you look into this one?

garypretty commented on libraries/Bot.Builder.Community.TranscriptStore.Elasticsearch/README.md:
```diff
@@ -0,0 +1,66 @@
+﻿# Bot.Builder.Community.TranscriptStore.Elasticsearch
```
Can you get the readme into the same format as the other storage provider (which I updated to be consistent with the rest of the repo). Thanks.

pratikpanda replied:
Sure, I will update the readme.

pratikpanda commented:
@garypretty, I have updated the README and fixed the project type for the test projects as well. Just a note that these are integration tests and therefore are dependent on Elasticsearch server. I am skipping these tests as part of the build process.
